app/src/main/java/org/mozilla/fenix/components/metrics/LeanplumMetricsService.kt

```diff
@@ -101,6 +101,15 @@ class LeanplumMetricsService(private val application: Application) : MetricsServ
 
             val installedApps = MozillaProductDetector.getInstalledMozillaProducts(application)
 
+            val trackingProtection = application.settings().run {
+                when {
+                    !shouldUseTrackingProtection -> null
```

Comment:
It may be fine, but I just want to make sure that Leanplum properly handles a "null" case here for when we data process this? I'm not tried adding a null value before.

Reply:
Renamed to String `none`, and updated the docs.